src/scripts/constants.ts

```diff
@@ -445,4 +447,9 @@ export module Constants {
 		export var pdfPageSelection = "pdfPageSelection";
 		export var serifGroupSet = "serifGroupSet";
 	}
+
+	export module AriaNavigation {
+		export var horizontal = "horizontal";
```

Comment:
See above comment on passing in the ariaSetDirection as a string. `string` is too broad of a type, and doesn't stop a dev from typing in the wrong thing.